lib/spack/spack/cmd/unit_test.py

```diff
@@ -217,6 +217,9 @@ def unit_test(parser, args, unknown_args):
             if pytest is None:
                 spack.bootstrap.ensure_environment_dependencies()
                 import pytest
+    else:
```

Comment:
@danlipsa If you look at line 213 you can see the reason we skip this is because at the time Clingo was not supported via bootstrap on Windows. This is no longer the case, so we can actually just be rid of this entire if/else stanza.

Reply:
Actually, I could not bootstrap cling. I installed it using pip.

Reply:
@danlipsa  can you coordinate with @johnwparent about getting this working for you if this is still a problem?